README.md

```diff
@@ -7,3 +7,145 @@ Derived from IDS Usage Control App by Fraunhofer IESE
 [![Docker badge](https://img.shields.io/docker/pulls/rdlabengpa/ids_uc_data_app.svg)](https://hub.docker.com/r/rdlabengpa/ids_uc_data_app)
 <br/>
 
+The TRUE Connector Usage Control Data App, based on IDS Usage Control App by Fraunhofer IESE enabling:
+
+- provides access to a resource with specific identifier defined by the target property or rule in a specific time interval
+
+## How to Configurate
+
+The configuration should be performed in the application.yml file:
+
+- set the port 
+	server:
+     port: 9555
+	 
+- save the policies to the file storage(If you are using a persistance database to store the policies set it to false. (Default = true))
+    application:
+	 savePoliciesToFilestorage: true
+	 
+- add the path of your directory where you want to save the policies:
+	odrlPolicyDirectory:
+
+## Run the application and import rules
+
+UsageControl dataApp exposes endpoint for importing rules. Following endpoint can be found at:
+   
+   http://localhost:9555/swagger-ui.html
+   
+  
+On the following screenshots, can be found instructions how to import rules:
```

Comment:
"can be found" ?
Maybe On the following screenshots instructions for importing rule are depicted.